examples/exti.rs

```diff
@@ -61,8 +60,7 @@ fn main() -> ! {
         int_pin.enable_interrupt(&p.EXTI);
     } // initialization ends here
 
-    let mut nvic = cp.NVIC;
-    nvic.enable(pac::Interrupt::EXTI9_5);
+    unsafe { pac::NVIC::unmask(pac::Interrupt::EXTI9_5); }
```

Comment:
More unsafe? :-(

Reply:
https://docs.rs/cortex-m/0.6.2/cortex_m/peripheral/struct.NVIC.html#method.enable

deprecated

Reply:
OK, thanks for the link.